.github/workflows/unittests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       matrix:
         os: [macos-13, ubuntu-latest]
         python-version: ["3.8", "3.12"]
-        bitcoind-version: ["0.18.0", "27.1"]
+        bitcoind-version: ["0.22.0", "27.1"]
```

Comment:
v22 was the first version where the notation went from v0.x to vx.0
so that should be changed too